src/synthesizer/abundances.py

```diff
@@ -245,118 +326,137 @@ class Elements:
 
 class Abundances(Elements):
     def __init__(
-        self, Z=Elements.Z_sol, alpha=0.0, C="Dopita2006", N="Dopita2006", d2m=False
+        self,
+        metallicity=Elements.solar_metallicity,
+        alpha=0.0,
+        abundances=False,
+        carbon_abundance=False,
```

Comment:
Could you add these args to the docstring

Reply:
I'm confused, they are there aren't they?